client/components/deposits-overview/style.scss

```
@@ -41,7 +40,9 @@
 				}
 			}
```

I think this will do the trick for adding margin-bottom to a single deposit:
```suggestion
			&:last-child {
				margin-bottom: 16px;
			}
		}
```